External/FEXCore/Source/Interface/Config/Config.cpp

```diff
@@ -352,7 +352,7 @@ namespace JSON {
       // Only return if it exists
       std::error_code ec{};
       if (std::filesystem::exists(Path, ec)) {
-        return Path;
+        return Path.c_str();
```

Comment:
This and the next comment about application names gets resolved in the next PR, so I'm hesitant to change again since merge conflicts are a pain with these.

Reply:
This pattern recurs a *lot* among the 126 matches for `c_str()` in this PR. Since we've opened *several* PRs very recently to remove performance overhead from unnecessary string/path copies, having a controlled way of adding new ones seems desirable.

I think an easy way to achieve this with reasonable effort would be patch up this PR with one "add fextl::string_from_path" commit and rebase the next PR on a partial revert of that same patch. That way you don't need to do resolve any merge conflicts but review becomes easier.

Reply:
Converted the c_str usage over to helpers that will be removed in the next PR.